### PR TITLE
chore(printers): add unit test coverage for printers part

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- add unit test coverage for printers part
#### Which issue(s) this PR fixes:

Fixes # None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by googs1025** [comment]
@ardaguclu 
can you help this for me :)

File: pkg/printers/internalversion/printers_test.go [hunk 1]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+func TestPrintClusterTrustBundle(t *testing.T) {
+	tests := []struct {
+		bundle   certificates.ClusterTrustBundle
+		options  printers.GenerateOptions
+		expected []metav1.TableRow
+	}{
+		{
+			bundle: certificates.ClusterTrustBundle{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "k8s.io:foo:",
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6787]: Does adding `CreationTimestamp` make sense?

File: pkg/printers/internalversion/printers_test.go [hunk 2]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+				},
+				Spec: certificates.ClusterTrustBundleSpec{
+					SignerName: "k8s.io/foo",
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6790]: Could you please add a better name Signer and Object name?

File: pkg/printers/internalversion/printers_test.go [hunk 3]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+		expected                         []metav1.TableRow
+	}{
+		{
+			validatingAdmissionPolicyBinding: admissionregistration.ValidatingAdmissionPolicyBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "config",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(-3e11)},
+				},
+				Spec: admissionregistration.ValidatingAdmissionPolicyBindingSpec{
+					PolicyName: "xyzlimit-scale.example.com",
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6828]:
```suggestion
					PolicyName: "test.example.com",
```

File: pkg/printers/internalversion/printers_test.go [hunk 4]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+					ParamRef: &admissionregistration.ParamRef{
+						Name:                    "xyzlimit-scale-setting.example.com",
+						ParameterNotFoundAction: ptr.To(admissionregistration.DenyAction),
+					},
+					MatchResources: &admissionregistration.MatchResources{
+						MatchPolicy: func() *admissionregistration.MatchPolicyType {
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6834]: This is not used. Why are we setting this in here?

File: pkg/printers/internalversion/printers_test.go [hunk 5]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+func TestPrintLeaseCandidate(t *testing.T) {
+	tests := []struct {
+		leaseCandidate coordination.LeaseCandidate
+		options        printers.GenerateOptions
+		expected       []metav1.TableRow
+	}{
+		{
+			leaseCandidate: coordination.LeaseCandidate{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "test-leaseCandidate",
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6872]:
```suggestion
					Name:              "test-lease",
```

File: pkg/printers/internalversion/printers_test.go [hunk 6]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(-3e11)},
+				},
+				Spec: coordination.LeaseCandidateSpec{
+					BinaryVersion:       "test-binary-version",
+					EmulationVersion:    "test-emulation-version",
+					LeaseName:           "test",
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6878]:
```suggestion
					LeaseName:           "test-lease-name",
```

File: pkg/printers/internalversion/printers_test.go [hunk 7]
```diff
@@ -6775,6 +6775,132 @@ func TestPrintScale(t *testing.T) {
...
+					PreferredStrategies: []coordination.CoordinatedLeaseStrategy{coordination.OldestEmulationVersion},
```
Comment (ardaguclu) [inline pkg/printers/internalversion/printers_test.go:6879]: Is this used in this test?

File: pkg/printers/internalversion/printers_test.go (hunk 4 above)
Comment (googs1025) [reply pkg/printers/internalversion/printers_test.go]: removed

File: pkg/printers/internalversion/printers_test.go (hunk 7 above)
Comment (googs1025) [reply pkg/printers/internalversion/printers_test.go]: removed

File: pkg/printers/internalversion/printers_test.go (hunk 1 above)
Comment (googs1025) [reply pkg/printers/internalversion/printers_test.go]: This resource object does not use the createTime field

**Comment by googs1025** [comment]
/retest
retest for no-related flake

**Comment by ardaguclu** [comment]
Thank you
/lgtm
/approve

**Pull request merged**